OpenTabletDriver.UX/Windows/Bindings/BindingEditorDialog.cs

```diff
@@ -80,5 +85,97 @@ private static string ParseMouseButton(MouseEventArgs e)
                     return nameof(MouseButton.None);
             }
         }
+
+        private class BindingController : TextArea
+        {
+            public BindingController()
+            {
+                TextAlignment = TextAlignment.Center;
+                ReadOnly = true;
+                ToolTip = TOOLTIP;
+                Text = TOOLTIP;
+            }
+
+            private const string TOOLTIP = "Press a key, combination of keys, or a mouse button.";
+
+            private PluginSettingStore store;
+            public PluginSettingStore Store
+            {
+                set
+                {
+                    this.store = value;
+                    Refresh();
+                }
+                get => this.store;
+            }
+
+            public void Refresh()
+            {
+                this.Text = Store?.GetHumanReadableString() ?? TOOLTIP;
+            }
+
+            protected override void OnKeyDown(KeyEventArgs e)
+            {
+                PluginSettingStore store;
+                Keys keys = e.KeyData;
+
+                if (keys == Keys.None)
+                    return;
+
+                if (keys.HasFlag(Keys.Control | Keys.LeftControl) || keys.HasFlag(Keys.Control | Keys.RightControl))
+                    keys &= ~Keys.Control;
+                if (keys.HasFlag(Keys.Alt | Keys.LeftAlt) || keys.HasFlag(Keys.Alt | Keys.RightAlt))
+                    keys &= ~Keys.Alt;
+                if (keys.HasFlag(Keys.Shift | Keys.LeftShift) || keys.HasFlag(Keys.Shift | Keys.RightShift))
+                    keys &= ~Keys.Shift;
+                if (keys.HasFlag(Keys.Application | Keys.LeftApplication) || keys.HasFlag(Keys.Application | Keys.RightApplication))
+                    keys &= ~Keys.Application;
```

Comment:
```suggestion
                if (keys.HasFlag(Keys.Alt | Keys.LeftAlt) || keys.HasFlag(Keys.Alt | Keys.RightAlt))
                    keys &= ~Keys.Alt;
                else if (keys.HasFlag(Keys.Control | Keys.LeftControl) || keys.HasFlag(Keys.Control | Keys.RightControl))
                    keys &= ~Keys.Control;
                else if (keys.HasFlag(Keys.Shift | Keys.LeftShift) || keys.HasFlag(Keys.Shift | Keys.RightShift))
                    keys &= ~Keys.Shift;
                else if (keys.HasFlag(Keys.Application | Keys.LeftApplication) || keys.HasFlag(Keys.Application | Keys.RightApplication))
                    keys &= ~Keys.Application;
```
This fixes the sequence issues. Modifying `Control` before `Alt` leads to a key deletion so check it first, and delete only once to prevent the key deletion due to the next `Control` check (the duplicate only happens once)
> `HasFlag(LeftControl)` also satisfies `HasFlag(LeftAlt)` when the key is `LeftControl`